tests/app/main/test_frameworks.py

```diff
@@ -1197,7 +1197,7 @@ def _assert_clarification_email(self, send_email, is_called=True, succeeds=True)
                 "MANDRILL",
                 "Test Framework clarification question",
                 "suppliers+g-cloud-7@digitalmarketplace.service.gov.uk",
```

Comment:
Should check with Nicole what email address she wants this to come from so her filters work.